pimd/pim_cmd.c

```diff
@@ -7193,12 +7193,12 @@ DEFUN (ip_pim_joinprune_time,
 
 DEFUN (no_ip_pim_joinprune_time,
        no_ip_pim_joinprune_time_cmd,
-       "no ip pim join-prune-interval (5-600)",
+       "no ip pim join-prune-interval [(1-65535)]",
        NO_STR
        IP_STR
        "pim multicast routing\n"
        "Join Prune Send Interval\n"
-       "Seconds\n")
+       IGNORED_IN_NO_STR)
 {
 	char jp_default_timer[5];
```

Comment:
This buffer will not work if the default ever becomes 5 digits... Can't we stringnify the `define` so we don't need any stack buffers? Or maybe call `DESTROY` instead of `MODIFY`.

I know this is not related with your change, but just a suggestion (I won't block you for it).

Reply:
changed